EASI.postman_collection.json

```diff
@@ -1178,8 +1178,8 @@
 								"body": {
 									"mode": "graphql",
 									"graphql": {
-										"query": "mutation createSystemIntakeGRBDiscussion($input: createSystemIntakeGRBDiscussionPostInput!) {\n  createSystemIntakeGRBDiscussionPost(input: $input) {\n    id\n    content\n    grbRole\n    votingRole\n    systemIntakeID\n    createdByUserAccount {\n        id\n        username\n    }\n  }\n}",
-										"variables": "{\r\n  \"input\": {\r\n    \"systemIntakeID\": \"8edb237e-ad48-49b2-91cf-8534362bc6cf\",\r\n    \"content\": \"<p>banana apple carburetor</p>\"\r\n  }\r\n}"
+										"query": "mutation createSystemIntakeGRBDiscussion(\n  $input: createSystemIntakeGRBDiscussionPostInput!\n) {\n  createSystemIntakeGRBDiscussionPost(input: $input) {\n    id\n    content\n    grbRole\n    votingRole\n    systemIntakeID\n    createdByUserAccount {\n      id\n      username\n    }\n  }\n}\n",
+										"variables": "{\r\n  \"input\": {\r\n    \"systemIntakeID\": \"8edb237e-ad48-49b2-91cf-8534362bc6cf\",\r\n    \"content\": \"<p>banana apple carburetor Let me look into it, ok? <span data-type=\\\"mention\\\" tag-type=\\\"USER_ACCOUNT\\\" class=\\\"mention\\\" data-id-db=\\\"8dceda-be23-4822-aa69-a3f67de6078b\\\">@Audrey Abrams</span>!\\\"</p>\"\r\n  }\r\n}"
```

Comment:
Would you be able to either:

1. Add to this `content` variable to tag the other group types
2. Make copies of this request that tag the other group types

I think that'd be helpful for testing the behavior of what to do on each tag type when this gets merged (and just making sure those tags actually get parsed correctly on the BE!)

Reply:
yes! will add - thanks

Reply:
added!